### PR TITLE
Fix issue of undefined content in meldemichel gfi on narrow devices

### LINKED ISSUE #62: Mobile GFI window may show empty content
**Describe the bug**
It is possible to view "empty" GFI windows.

**Expected behavior**
The GFI window is supposed to close and switch to the FeatureList.

**Steps to reproduce**
1. Open client on mobile/narrow device
2. Open GFI by clicking a marker
3. Click into an empty region of the map
4. The GFI window now shows "empty" feature information.

**Screenshots**
gfi_breakable

### PR DESCRIPTION
**Pull request by dopenguin**
## Summary

See #62 for the description of the issue and how to reproduce it.

## Relevant tickets, issues, et cetera

Resolves #62

### CODE REVIEW

**Review by warm-coolguy — CHANGES_REQUESTED** [review]
Missing a changelog entry.

🏓 @dopenguin

File: packages/clients/meldemichel/src/plugins/Gfi/Feature.vue [hunk 1]
```diff
@@ -73,17 +73,29 @@ export default Vue.extend({
   computed: {
     ...mapGetters(['hasSmallWidth', 'hasWindowSize']),
     ...mapGetters('plugin/gfi', [
+      'gfiConfiguration',
       'imageLoaded',
+      'renderMoveHandle',
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/clients/meldemichel/src/plugins/Gfi/Feature.vue]: Done 00e30815

File: packages/clients/meldemichel/src/plugins/Gfi/Feature.vue [hunk 2]
```diff
@@ -73,17 +73,29 @@ export default Vue.extend({
   computed: {
     ...mapGetters(['hasSmallWidth', 'hasWindowSize']),
     ...mapGetters('plugin/gfi', [
+      'gfiConfiguration',
```
[the remark this answers is not in the document]
Comment (dopenguin) [reply packages/clients/meldemichel/src/plugins/Gfi/Feature.vue]: Done 00e30815

**Comment by dopenguin** [comment]
> Missing a changelog entry.
> 
> 🏓 @dopenguin

Added a44be908

🏓 @warm-coolguy

**Review by warm-coolguy — APPROVED** [review]
🚀 @dopenguin

**Pull request merged**